zebra/zebra_rib.c

```diff
@@ -2021,9 +2022,9 @@ static void rib_process_result(struct zebra_dplane_ctx *ctx)
 	rn = rib_find_rn_from_ctx(ctx);
 	if (rn == NULL) {
```

Comment:
in these two, maybe it's not worth including the `%pRN` at all - since the `rn` is NULL?